olm/olm.sh

```diff
@@ -112,30 +145,20 @@ installOperatorMarketPlace() {
       echo "Catalog source not created after 2 minutes"
       exit 1
     fi
-
-    kubectl get catalogsource/"${packageName}" -n "${marketplaceNamespace}" -o json | jq '.metadata.namespace = "olm" | del(.metadata.creationTimestamp) | del(.metadata.uid) | del(.metadata.resourceVersion) | del(.metadata.generation) | del(.metadata.selfLink) | del(.status)' | kubectl apply -f -
```

Comment:
Why did you move these instructions at a place where `applyCheOperatorSource` hasn't been applied already ?

Reply:
@davidfestal Sorry was an error from my side.